src/util/filter/registerFKSelectorFilter.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import {registerCustomFilter} from "./CustomFilter";
+import {registerCustomFilterRenderer} from "./CustomFilterRenderer";
+import {field, value} from "../../../filter";
+import {Field, useFormConfig} from "../../../../domainql-form";
+import {FKSelector} from "../../index";
+
+
+/**
+ * Register a filter renderer for an FKSelector
+ *
+ * @param {String} name             filter renderer alias
+ * @param {Object} query            the query used for the FKSelector
+ * @param {String} rootType         the table that the FKSelector works on
+ * @param {String} sourceName       path of the FKSelector inside the table
+ * @param {String} modalTitle       title of the FKSelectorModal
+ * @param {String} valueFieldName   the value of the FKSelector table used by the filter
```

Comment:
Ist das so in unserem Code-Formatter drin? (Bezieht sich auch auf die Leerzeichen oben)